ocaml/testsuite/tests/typing-layouts-or-null/basics.ml

```diff
@@ -112,7 +113,8 @@ Error: Record element types must have a representable layout.
        The layout of t_value_or_null is value_or_null, because
```

Comment:
This is a test whose output was modified in the last three commits that touched a bunch of tests. Based on how GitHub is showing your commend to me, I think you were looking at a diff that doesn't include those changes when you left this comment. Just to make sure we are on the same page, the new error message is:
```
Error: Record element types must have a representable layout.
       The kind of t_value_or_null is value_or_null
         because of the definition of t_value_or_null at line 3, characters 0-36.
       But the kind of t_value_or_null must be a subkind of value
         because it is the type of record field x,
         defaulted to kind value.
```
Excluding the first line, this talks about kinds instead of layout, which is an improvement in my book. But I agree the first line is still bad. I'll see if I can improve it.

The `scannable` idea is interesting, althought I think its name would be confusing if we add a `scannability` axis.

Reply:
I think this might also be fine to leave for now - at least this case. If I understand correctly, this should be accepted in the future, as `value_or_null` should be considered representable.

Reply:
Yes, thanks for clarifying. I agree this is fine.